src/main/java/site/sonisori/sonisori/auth/jwt/JwtUtil.java

```diff
@@ -120,14 +124,41 @@ public boolean validateAccessToken(String accessToken) {
 		}
 	}
 
+	public boolean validateRefreshToken(String refreshToken) {
+		try {
+			UUID.fromString(refreshToken);
+
+			RefreshToken token = refreshTokenRepository.findById(refreshToken)
+				.orElseThrow(() -> new JwtException(ErrorMessage.NOT_FOUND_TOKEN.getMessage()));
+
+			return true;
+		} catch (IllegalArgumentException e) {
+			throw new IllegalArgumentException(ErrorMessage.INVALID_TOKEN.getMessage());
+		}
+	}
```

Comment:
저라면 이렇게 했을 것 같네용 굳이 getUserId라는 함수가 필요할까요???
```suggestion
	public void validateRefreshToken(String refreshToken, Long userId) {
    RefreshToken token = refreshTokenRepository.findById(refreshToken)
        .orElseThrow(() -> new JwtException(ErrorMessage.NOT_FOUND_TOKEN.getMessage()));

    // 요청된 userId와 저장된 userId가 일치하는지 확인
    if (!token.getUserId().equals(userId)) {
        throw new JwtException(ErrorMessage.NOT_FOUND_TOKEN.getMessage());
    }
}
```